_component_design/chips.md

```diff
@@ -17,27 +17,63 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Introduction
-
 <div class="hxRow" markdown="1">
 
-{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## When to use
 
-### When to use
+Use a chips to represent graphically a block of information. For example, use chips to represent a user or device.
 
-Chips are best used with indicating a list of items that may be part of a different workflow. For example, adding or removing a list of requestors on a ticket.
+## Best practices
 
-### Best practices
+Consider the following best practices when constructing a chip:
 
 - Avoid using chips in multiple locations on a page.
-- Ensure the user can concretely identify the item represented by the chip.
+- Label the chip so that the user can identify the item represented by the chip.
+
+{% endcolumn %}
+
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+
+## Specifications
+Refer to the following specifications when constructing chips.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Composition
+A chip is a single circle that contains a preview of the item the chip represents. The identification information can consist of the following:
```

Comment:
"Item" is used again here to represent the information found in the chip. If we're gonna use item to refer to the information, use that term consistently throughout. Or just stick with "information" instead of item.